src/report.ts

```diff
@@ -3,18 +3,31 @@ import { utcFormat } from "d3-time-format";
 const yearFormat = utcFormat("%Y");
 const yearMonthDayFormat = utcFormat("%Y-%m-%d");
 
+function domain(env: string): string {
+  switch (env) {
+    case "local":
+      return "";
+    case "prod":
+    case "staging":
+    case "dm":
+      return `https://public-reporting-data.${env}.login.gov`;
+    default:
+      return `https://public-reporting-data.${env}.identitysandbox.gov`;
```

Comment:
Feels a bit sketchy to put an unfiltered query parameter here. Maybe make `local` behavior the default and safelist the sandbox environments like we're doing with the production environments?

Reply:
that means that every time one of us adds our own personal sandbox environment, we have to safelist it?

Reply:
this also is only for the sanbox environment, so 1) there shouldn't be any PII and 2) errors are usually more ok?

Reply:
At the very least could we sanitize it, e.g. `env.replace(/\W/g, '')` ?

Example abuse I'm worried about: `?env=evil.example.com/a`

Reply:
Love it, done in 538b0879cb3ff9d468739e9e32a8f6d5082fb3f9

Reply:
Also raises a question of whether we could set up a CSP `connect-src` for the domains we're expecting to fetch from.

Reply:
That CSP header would have to come from Federalist, and looks like the only custom headers are Cache-Control right now? 

https://federalist.18f.gov/documentation/custom-headers/

Reply:
We can do it with a `<meta>` tag too.

https://developers.google.com/web/fundamentals/security/csp#the_meta_tag

Reply:
awesome, added in be38bc42222c0dfddd0fb820ba68802d47f8fd97, I had to remove the param sanitization but CSV did error as expected with `?env=evil.com/a`